plugins/PrivacyManager/templates/privacySettings.twig

```diff
@@ -7,266 +7,276 @@
     <p>{{ 'PrivacyManager_Teaser'|translate('<a href="#anonymizeIPAnchor">',"</a>",'<a href="#deleteLogsAnchor">',"</a>",'<a href="#optOutAnchor">',"</a>")|raw }}
         {{'PrivacyManager_SeeAlsoOurOfficialGuidePrivacy'|translate('<a href="http://piwik.org/privacy/" rel="noreferrer"  target="_blank">','</a>')|raw }}</p>
+
     <h2 id="anonymizeIPAnchor">{{ 'PrivacyManager_UseAnonymizeIp'|translate }}</h2>
     <form method="post" action="{{ {'action':'saveSettings', 'form':'formMaskLength', 'token_auth':token_auth} | urlRewriteWithParameters }}" id="formMaskLength">
-        <div id='anonymizeIpSettings'>
-            <table class="adminTable" style='width:800px;'>
-                <tr>
-                    <td width="250">{{ 'PrivacyManager_UseAnonymizeIp'|translate }}<br/>
-                        <span class="form-description">{{ 'PrivacyManager_AnonymizeIpDescription'|translate }}</span>
-                    </td>
-                    <td width='500'>
-                        <input id="anonymizeIPEnable-1" type="radio" name="anonymizeIPEnable" value="1" {% if anonymizeIP.enabled == '1' %}checked {% endif %}/>
-                        <label for="anonymizeIPEnable-1">{{ 'General_Yes'|translate }}</label>
-                        <input class="indented-radio-button" id="anonymizeIPEnable-0" type="radio" name="anonymizeIPEnable" value="0" {% if anonymizeIP.enabled == '0' %} checked {% endif %}/>
-                        <label for="anonymizeIPEnable-0">{{ 'General_No'|translate }}</label>
-                        <input type="hidden" name="token_auth" value="{{ token_auth }}"/>
-                    </td>
-                    <td width="200">
-                        <div style="width:180px">
-                            {{ piwik.inlineHelp('PrivacyManager_AnonymizeIpInlineHelp'|translate) }}
-                        </div>
-                    </td>
-                </tr>
-            </table>
+        <div id="anonymizeIpSettings" class="form-group">
+            <label>
+                {{ 'PrivacyManager_UseAnonymizeIp'|translate }}<br/>
+                <span class="form-description">{{ 'PrivacyManager_AnonymizeIpDescription'|translate }}</span>
+            </label>
+            <div class="form-help">
+                {{ 'PrivacyManager_AnonymizeIpInlineHelp'|translate }}
+            </div>
+            <label class="radio">
+                <input id="anonymizeIPEnable-1" type="radio" name="anonymizeIPEnable" value="1" {% if anonymizeIP.enabled == '1' %}checked {% endif %}/>
+                {{ 'General_Yes'|translate }}
+            </label>
+            <label class="radio">
+                <input class="indented-radio-button" id="anonymizeIPEnable-0" type="radio" name="anonymizeIPEnable" value="0" {% if anonymizeIP.enabled == '0' %} checked {% endif %}/>
+                {{ 'General_No'|translate }}
+            </label>
+            <input type="hidden" name="token_auth" value="{{ token_auth }}"/>
```

Comment:
Token is also set again here and in the form? Ideally we would maybe replace it by an `nonce` but that's not part of redesign :)